src/vector/init.tsx

```diff
@@ -115,6 +115,16 @@ export async function loadLanguage() {
     }
 }
 
+export async function loadSpellCheckLanguages() {
+    const langs = SettingsStore.getValue("spell-check-languages", null, false);
```

Comment:
As with the other layers, I'm a bit unsure whether we need the settings on top of Electron's own storage.

Reply:
I don't think we do since we have  [`getSpellCheckerLanguages()`](https://www.electronjs.org/docs/api/session#sesgetspellcheckerlanguages). The month or two younger me just didn't think of it :smile: 

I'll try to rework this